src/modules/Packages.rb

```diff
@@ -927,23 +936,66 @@ def architecturePackages
       deep_copy(packages)
     end
 
-    # Compute special packages
-    # @return [Array](string)
-    def modePackages
-      tags = []
-      tags << "sbl" if Linuxrc.braille
-      # ssh installation
+    # Checking if all needed packages for remote installation
+    # will be installed on the target system.
+    # @return [String] empty string or error message if there are missing packages
+    def check_remote_installation_packages
+      @missing_remote_kind = []
+      @missing_remote_packages = []
+
+      if Linuxrc.braille
+        missing = braille_packages.reject { |tag| pkg_will_be_installed(tag) }
+        unless missing.empty?
+          @missing_remote_packages << missing
+          @missing_remote_kind << "braille"
+        end
+      end
       if Linuxrc.usessh
-        # "ip" tool is needed by the YaST2.ssh start script (bnc#920175)
-        tags.concat(["openssh", "iproute2"])
+        missing = ssh_packages.reject { |tag| pkg_will_be_installed(tag) }
+        unless missing.empty?
+          @missing_remote_packages << missing
+          @missing_remote_kind << "ssh"
```

Comment:
As this is also displayed in the UI, using the upper case variant is better: `SSH`, `VNC`,...